dracut/modules.d/90qubes-pciback/qubes-pciback.sh

```diff
@@ -1,25 +1,57 @@
-#!/bin/sh
+#!/bin/bash --
 
 type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
+unset re HIDE_PCI usb_auth_default usb_in_dom0 dev skip exposed
 
-# Find all networking devices currenly installed...
-HIDE_PCI="`lspci -mm -n | grep '^[^ ]* "02'|awk '{print $1}'`"
+usb_auth_default=/sys/module/usbcore/parameters/authorized_default
+usb_in_dom0=false
 
-# ... and optionally all USB controllers...
 if getargbool 0 rd.qubes.hide_all_usb; then
-    HIDE_PCI="$HIDE_PCI `lspci -mm -n | grep '^[^ ]* "0c03'|awk '{print $1}'`"
+    # Select all networking and USB devices
+    re='0(2|c03)'
+elif ! getargbool 1 usbcore.authorized_default; then
+    # Select only networking devices, but enable USBguard
+    re='02'
+    systemctl start usbguard.service
+else
+    re='02'
+    warn 'USB in dom0 is not restricted'
 fi
```

Comment:
This needs to be a bit more robust. Specifically, enable usbguard in case of `rd.qubes.hide_all_usb usbcore.authorized_default=0 rd.qubes.dom0_usb=<some-specific-controller>`